test/tests/utils/spline_interpolation/tests

```diff
@@ -1,8 +1,13 @@
 [Tests]
-  [./test]
+  [./regular_spline]
     type = 'Exodiff'
     input = 'spline_interpolation.i'
     exodiff = 'spline_interpolation_out.e'
   [../]
+  [./bicubic_spline]
+    type = 'Exodiff'
+    input = 'bicubic_spline_interpolation.i'
+    exodiff = 'bicubic_spline_interpolation_out.e'
+    custom_cmp = 'bicubic_spline_interpolation.cmp'
+  [../]
 []
-  
```

Comment:
I think this is a github error. It's absolutely true that I removed a **blank** line from this file, and I can restore it if you want. But my file ends on a newline character. Here's the file viewed in whitespace-mode:

![i_have_newline_i_swear](https://cloud.githubusercontent.com/assets/10248304/20727116/d6673e7a-b63d-11e6-8fec-dac04706dc3b.png)

If I remove the new-line, here's the `git diff`:

![remove_newline](https://cloud.githubusercontent.com/assets/10248304/20727143/f71930d8-b63d-11e6-80b5-da6893a9811c.png)

indicating that there *was* a newline at the end of the file before I removed it. Moreover I've had this in my .emacs configuration file for months:

```
(add-hook 'before-save-hook 'delete-trailing-whitespace)
```